inc/saitunnel.h

```diff
@@ -451,7 +451,7 @@ typedef enum _sai_tunnel_attr_t
      * @type sai_object_id_t
      * @flags MANDATORY_ON_CREATE | CREATE_ONLY
      * @objects SAI_OBJECT_TYPE_ROUTER_INTERFACE
-     * @condition SAI_TUNNEL_ATTR_TYPE == SAI_TUNNEL_TYPE_IPINIP or SAI_TUNNEL_ATTR_TYPE == SAI_TUNNEL_TYPE_IPINIP_GRE
+     * @condition SAI_TUNNEL_ATTR_TYPE == SAI_TUNNEL_TYPE_IPINIP or SAI_TUNNEL_ATTR_TYPE == SAI_TUNNEL_TYPE_IPINIP_GRE or SAI_TUNNEL_ATTR_TYPE == SAI_TUNNEL_TYPE_VXLAN
```

Comment:
I have question here, what if we are using triditional l2 vxlan? then there is no underlay interface, but we can assume in that case we still need to supply underlay interface, but it will not bu used?

Reply:
How to account for VRF in the termination/decap?

Reply:
I think that even in traditional l2 vxlan, after encapsulation, the packet is than routed in L3 router, and thus still needs an underlay interface to determine underlay vrf.
No?

Reply:
Is HEAD END REPLICATION supported for VXLAN, Any SAI attributes are defined for them?